packages/interactor/src/index.ts

```diff
@@ -14,3 +14,5 @@ export { CheckBox } from './definitions/check-box';
 export { RadioButton } from './definitions/radio-button';
 export { Select } from './definitions/select';
 export { MultiSelect } from './definitions/multi-select';
+
+export * as CypressCommands from './integration/cypress';
```

Comment:
This module doesn't export anything, but instead is evaluated for its side-effects. The canonical method for doing this is like so:

```ts
import './integration/cypress'
```

That way it's clear to anyone reading the code that this think is going to tinker with global state.